backend/env.py

```diff
@@ -1,35 +1,8 @@
 from pathlib import Path
 
+ROOT_PATH = str(Path(__file__).parents[1]) + '/'
```

Comment:
I am not sure that the windows server will handle this path properly - if changing the paths, could you deploy the branch first to make sure it works? then we can proceede with the merge

Reply:
You are correct, this would probably not work because we transformed it into a string -> I changed it for using only `Path` objects (possible because we simply use the in pd.read_csv)